AEPMessaging/Sources/rules/JSONRulesParser.swift

```diff
@@ -41,7 +41,7 @@ class JSONRulesParser {
 struct JSONRuleRoot: Codable {
     var version: Int
     var rules: [JSONRule]
-    
+
     /// Converts itself to `LaunchRule` objects, which can be used in `AEPRulesEngine`
     /// - Returns: an array of `LaunchRule` objects
```

Comment:
not related to this PR.. But we can change all the `LaunchRule ` in the docs to `MessagingRule `